gallery-so/gallery
# display notification for first post by a user

### Summary of Changes
display notification for first post by a user. web + mobile

### Demo or Before/After Pics
web
<img width="434" alt="CleanShot 2023-11-15 at 15 17 44@2x" src="https://github.com/gallery-so/gallery/assets/80802871/6763328b-2422-44b0-ae24-fe4d90467449">


mobile
![CleanShot 2023-11-15 at 15 17 33](https://github.com/gallery-so/gallery/assets/80802871/e5911c6e-73c7-4ca3-8aff-599a690654a5)
![CleanShot 2023-11-15 at 15 17 53](https://github.com/gallery-so/gallery/assets/80802871/c7fac1b1-ba70-416a-b14a-caf94cfad457)



### Edge Cases
- caption vs no caption


### Testing Steps
- with user A, follow user B who has never posted before
- with user B, create a post
- with user A, confirm you see this notification

### Checklist

Please make sure to review and check all of the following:
